evap/staff/forms.py

```diff
@@ -183,9 +183,9 @@ class EvaluationForm(forms.ModelForm):
 
     class Meta:
         model = Evaluation
-        fields = ('course', 'name_de', 'name_en', 'weight', 'is_rewarded', 'is_midterm_evaluation',
-                  'vote_start_datetime', 'vote_end_date', 'participants', 'general_questionnaires',
-                  'last_modified_time', 'last_modified_user_name')
+        fields = ('course', 'name_de', 'name_en', 'weight', 'is_rewarded', 'allow_editors_to_edit',
```

Comment:
please move `allow_editors_to_edit` before `is_rewarded`

Reply:
Alrighty. For self reference, in case I need to rebuild the test data:

```python
import random
e = list(Evaluation.objects.filter(course__semester=21))
chosen = random.choices(e, k=int(len(e)*0.2))
ids = [e.id for e in chosen]
Evaluation.objects.filter(id__in=ids).update(allow_editors_to_edit=False)
```